fuse/data/visualizer/visualizer_base.py

```diff
@@ -1,5 +1,14 @@
 from abc import ABC, abstractclassmethod, abstractmethod
-from typing import Dict, Any, List
+from typing import Dict, Any, List ,Union , Tuple
+from fuse.utils.ndict import NDict
+from torch import Tensor
+from torchvision.utils import save_image
+import matplotlib
+import numpy as np
+import nibabel as nib
+import os
+from fuseimg.utils.typing.key_types_imaging import DataTypeImaging
```

Comment:
fuse.data can't import fuseimg
You shold probably move the imaging part to fuseimg

Reply:
this file is under fuseimg only now